lib/helper/Playwright.js

```diff
@@ -960,20 +960,27 @@ class Playwright extends Helper {
   /**
    * {{> dragAndDrop }}
    *
-   * @param {any} [options] [Additional options](https://playwright.dev/docs/api/class-page#page-drag-and-drop) can be passed as 3rd argument.
-   *
    * ```js
    * // specify coordinates for source position
    * I.dragAndDrop('img.src', 'img.dst', { sourcePosition: {x: 10, y: 10} })
    * ```
    *
-   * > By default option `force: true` is set
    */
-  async dragAndDrop(srcElement, destElement, options = { force: true }) {
-    const src = new Locator(srcElement, 'css');
-    const dst = new Locator(destElement, 'css');
+  async dragAndDrop(srcElement, destElement) {
```

Comment:
If I understand well, it would break my use case because you removed the options parameter:
```ts
    // Move the object from default to initial position
    I.dragAndDrop(konvajsFragment.gameArea, konvajsFragment.gameArea, {
        sourcePosition: OBJECT_DEFAULT_POSITION,
        targetPosition: OBJECT_INITIAL_POSITION,
    });
```

Reply:
@kobenguyent 
My use case is "Move an object from one position to another one inside an area (canvas)". Therefore I need to pass source and target coordinates to `I.dragAndDrop()`.